pmd-core/src/main/java/net/sourceforge/pmd/benchmark/TimingReport.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.benchmark;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentMap;
+
+import net.sourceforge.pmd.benchmark.TimeTracker.TimedOperation;
+import net.sourceforge.pmd.benchmark.TimeTracker.TimedResult;
+
+/**
+ * A report on timing data obtained through the execution of PMD
+ * @author Juan Martín Sotuyo Dodero
+ */
+public class TimingReport {
+
+    private final long wallClockMs;
+    private final ConcurrentMap<TimedOperation, TimedResult> results;
+    
+    /* package */ TimingReport(final long wallClock, final ConcurrentMap<TimedOperation, TimedResult> accumulatedResults) {
+        wallClockMs = wallClock;
```

Comment:
Curious why this class is the one which first labels the units on the wall clock time?  I would suggest adding a unit suffix to all the timing variables.  So `wallClockMs` everywhere.

For naming, I think `Millis` is more common than `Ms`.  For example `System.currentTimeMillis()` and `TimeUnit.toMillis`.

Also, consider using `Nanos` on the other temporal units.  Or, be agnostic in naming by including a `TimeUnit` with each quantity, and use `TimeUnit.convert(...)` to move between them.